iosdevapp/GenerateImageView.swift

```diff
@@ -11,60 +11,86 @@ struct GenerateImageView: View {
     @StateObject var viewModel = GenerateImageViewModel()
     
     var body: some View {
+        VStack {
+            headerView
+            bodyView
```

Comment:
nice that you started to split the view in subviews. Maybe we could take it to a next level and split it even more? So that the body gets more readable.